### DISCUSSION TITLE
Account sign in/up problem

### DISCUSSION THREAD

**Discussion by RNyaupane** [vercel/community#5031] [Account Issue] [answered]
### Question

**Why I can't login and create account to vercel??**

**Comment by amyegan** [comment]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

**Comment by RNyaupane** [comment]
I tried to continue signing up and signing in with github. It says "User
account is blocked". When I tried to sign in with my gmail I couldn't login
and when I try to create account it says "User account is blocked".

**Reply by amyegan** [reply, marked as answer]
Please open a case to get a more private line of communication with our Customer Success team. You can open a case by using the appropriate [direct channel](https://vercel.com/guides/how-to-get-vercel-support#direct-channels) found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)
